### ISSUE TITLE
BKCtl can not show the result

### ISSUE THREAD

**Issue by zymap** [apache/bookkeeper#2236]
**BUG REPORT**

When I using bkctl to get the auditor, there is nothing to show. But I can get the auditor using REST API.

Dive into the code, I found the result is output to the log.

```
BookieSocketAddress bookieId = AuditorElector.getCurrentAuditor(conf, zk);
if (bookieId == null) {
       LOG.info("No auditor elected");
       return false;
}
LOG.info("Auditor: " + getBookieSocketAddrStringRepresentation(bookieId));
```

***Expected behavior***

The command result should output to the console.

**Comment by eolivelli** [comment]
I totally agree.
can you please send a patch?

**Comment by zymap** [comment]
@eolivelli Sure. I will send a pull request later.

**Comment by eolivelli** [comment]
resolved with #2268
